content/10.diagnostics.md

```diff
@@ -1,33 +1,54 @@
 
 #### RT-PCR
 
+RT-PCR tests are a type of test which uses reverse transcriptase to translate the viral RNA into DNA for pre-processing and uses PCR testing for sequencing.
+There are different reagents used for library preparation that are specific to identifying one or more target section(s) with PCR  [@doi:10.1021/acsnano.0c02624].
 The Drosten Lab, from Germany, was the first lab to establish and validate a diagnostic test to detect SARS-CoV-2.
 This test uses real-time reverse transcription PCR (RT-PCR) [@doi:10.2807/1560-7917.ES.2020.25.3.2000045] to detect several regions of the viral genome: the orf1b of the RNA dependent RNA polymerase (RdRP), the Envelope protein gene (E), and the Nucleocapsid protein gene (N).
 In collaboration with several other labs in Europe and in China, the researchers confirmed the specificity of this test with respect to other coronaviruses against specimens from 297 patients infected with a broad range of respiratory agents.
 Specifically this tests utilizes two probes against RdRP of which one is specific to SARS-CoV-2 [@doi:10.2807/1560-7917.ES.2020.25.3.2000045].
 Importantly, this assay did not give any false positive results.
 
+
 #### qRT-PCR
 
-Around the same time, researchers in China developed a quantitative real-time reverse transcription PCR (qRT-PCR) test to identify two regions of the viral genome, orf1b and N region [@doi:10.1093/clinchem/hvaa029].
+Around the same time as the above RT-PCR tests, researchers in China developed a quantitative real-time reverse transcription PCR (qRT-PCR) test to identify two regions of the viral genome, orf1b and N region [@doi:10.1093/clinchem/hvaa029].
 Specifically, this assay was tested on samples coming from two COVID-19 patients, including a panel of positive and negative controls consisting of RNA extracted from several cultured viruses.
 The assay uses the N gene to screen patients, while the orf1b region is used to confirm the infection [@doi:10.1093/clinchem/hvaa029].
 In this case the test was designed to detect sarbecoviruses, of which SARS-CoV-2 is one.
 Considering that no other sarbecoviruses are currently known to infect humans, a positive test indicates that the patient is infected with SARS-CoV-2.
 However, this test is not able to discriminate the genetics of viruses within the sarbecovirus clade.
 
+
+#### Pooled and Automated PCR Testing
+
+Due to limited supplies and the need for more tests, several labs have found ways to enable pooled or similar tests.
+The first such result came from a group at the Technion, which found they could pool up to 32 samples [@doi:10.1101/2020.03.26.20039438].
+This was followed by larger-scale pooling utilizing slightly different methods at the University of Minnesota [@doi:10.1101/2020.04.02.022186].
+These approaches are PCR-based but allow for more rapid scaling and higher efficiency for testing.
```

Comment:
```suggestion
Although these approaches are also PCR based, they allow for more rapid scaling and higher efficiency for testing than the initial PCR-based methods developed.
```
Is this the correct "than what" that you meant?

Reply:
"PCR-based"